### ISSUE TITLE
MMRA when user navigates to a post

### ISSUE THREAD

**Issue by thebaer** [thebaer/MMRA#1]
Right now Medium is only Made Readable Again when a site first loads. This may leave readers with an un-great experience after they navigate to a post

**Comment by chris-morgan** [comment]
The proper solution for this is probably to detect Medium’s meta tag and then just inject a stylesheet via [chrome.tabs.insertCSS](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/insertCSS).

The actual CSS will be something along the lines of

```css
.metabar.u-fixed {
	position: relative !important;
}

.js-stickyFooter,
.js-postShareWidget,
footer>.container:first-child {
	display:none !important;
}
```

**Comment by thebaer** [comment]
Oh awesome -- thanks! Didn't know that was possible. Will try that out this evening.

**Comment by panphora** [comment]
Just ran into this and came here to report it. My first impression was that the extension just wasn't working at all because I navigated to a post from the Hacker Noon front page and the top nav bar was sticky. Would be great to have a fix for this.

**Comment by thebaer** [comment]
`chrome.tabs.insertCSS` needs a background script to communicate with the content script and run the function. #7 quickly fixed the issue for v1.2; the `insertCSS` solution has been moved to #8.
